docs/notifications.md

```diff
@@ -531,6 +537,37 @@ Example:
 }
 ```
 
+### `header_of_added_block` notification
+Notification is sent when new block is stored, which is happens after block's processing.
```

Comment:
Please, take a look at the format of `block_added` comment (and the rest of the comments). The comment to header notification should have the same format, i.e. start with `The first parameter (`params` section)...`. with all appropriate differences from verbose `getblock` response mentioned (the same differences as `block_added` mentions plus an absence of transactions).